hypervisor/common/ptdev.c

```diff
@@ -191,8 +192,9 @@ ptdev_deactivate_entry(struct ptdev_remapping_info *entry)
 
 void ptdev_init(void)
 {
-	if (get_cpu_id() > 0)
+	if (get_cpu_id() != BOOT_CPU_ID) {
```

Comment:
Hi Shiqing, Is the ptdev_init only called by the BSP? Can we use precondition instead of this checking? Please check it, thanks!

Reply:
@wuxyintel Yes, ptdev_init is only called by BSP.
@dongyaozu Eddie, do you agree that we should use precondition to replace this check?